routes/designerRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const designerService = require('../services/designerService');
+
+const designerRouter = express.Router();
+
+designerRouter.get('/', designerService.getAll);
+designerRouter.get('/:id', designerService.getDesignerById);
+designerRouter.post('/create', designerService.create);
+designerRouter.update('/update', designerService.update);
```

Comment:
I just realized that `Router.update` is not a thing....I think it's `Router.put`

I'll update this here and in productRouter.js